po/pt.po

```diff
@@ -0,0 +1,273 @@
+
+#: src/Application.vala:171
+msgid "Stretch your arms, and twist your wrists for 10 seconds."
+msgstr "Estique os seus braços, e torça os seus pulsos durante 10 segundos."
+
+#: src/Application.vala:172
+msgid "Arms:"
+msgstr "Braços:"
+
+#: src/Application.vala:177
+msgid "Stretch your legs"
+msgstr "Estique as suas pernas"
+
+#: src/Application.vala:178
+msgid "Stand up, twist each ankle, and bend each knee."
+msgstr "Levante-se, torça cada tornozelo, e dobre cada joelho."
+
+#: src/Application.vala:179
+msgid "Legs:"
+msgstr "Pernas:"
+
+#: src/Application.vala:184
+msgid "Turn your neck"
+msgstr "Vire o seu pescoço"
+
+#: src/Application.vala:185
+msgid "Turn your head in all directions. Repeat 3 times."
+msgstr "Vire a sua cabeça em todas as direções. Repita 3 vezes."
+
+#: src/Application.vala:186
+msgid "Neck:"
+msgstr "Pescoço:"
+
+#: src/Application.vala:191
+msgid "Watch your posture"
+msgstr "Veja a sua postura"
+
+#: src/Application.vala:192
+msgid "Make sure your back is straight."
+msgstr "Certifique-se que tem as costas direitas."
+
+#: src/Application.vala:193
+msgid "Posture:"
+msgstr "Postura:"
+
+#: src/MainGrid.vala:59
+msgid "Reminders"
+msgstr "Lembretes"
+
+#: src/MainGrid.vala:74
+msgid "Decide how often Badger should remind you to relax these:"
+msgstr "Decida com que frequência o Badger o deve lembrar para relaxar:"
+
+#: src/MainGrid.vala:114
+msgid "1 min"
+msgstr "1 min"
+
+#: src/MainGrid.vala:116
+msgid "30 min"
+msgstr "30 min"
+
+#: src/MainGrid.vala:118
+msgid "1 hour"
+msgstr "1 hora"
+
+#: src/MainGrid.vala:149
+#, c-format
+msgid "%.0f min"
+msgstr "%.0f min"
+
+#: data/com.github.elfenware.badger.desktop.in:4
+#: data/com.github.elfenware.badger.appdata.xml.in:7
+msgid "Badger"
+msgstr "Badger"
+
+#: data/com.github.elfenware.badger.desktop.in:5
+msgid "Ergonomic Reminder"
+msgstr "Lembrete Ergonómico"
+
+#: data/com.github.elfenware.badger.desktop.in:6
+msgid "Remind yourself to relax your eyes and limbs"
+msgstr "Lembre-se a si mesmo para relaxar aos seus olhos e membros"
+
+#: data/com.github.elfenware.badger.desktop.in:8
+msgid "com.github.elfenware.badger"
+msgstr "com.github.elfenware.badger"
+
+#: data/com.github.elfenware.badger.desktop.in:11
+msgid "Ergonomics;Health;Eyes;Muscles;"
+msgstr "Ergonomia;saúde;olhos;músculos;"
+
+#: data/com.github.elfenware.badger.appdata.xml.in:8
+msgid "Remind yourself to not sit and stare at the screen for too long"
+msgstr ""
+"Lembre-se a si mesmo para não ficar sentado e olhar para o ecrã durante "
+"muito tempo"
+
+#: data/com.github.elfenware.badger.appdata.xml.in:10
+msgid ""
+"Whether you're dashing through Mario Kart on MupenGUI, binge-watching "
+"Silicon Valley on Cinema, or writing your next novel on Quilter, you need to "
+"relax your body every once in a while."
+msgstr ""
+"Se você estiver a correr no Mario Kart no MupenGUI, a ter uma maratona de "
+"cinema em Silicon Valley, ou a escrever o seu próximo romance no Quilter, "
```

Comment:
I'm not sure about this line. Here, Cinema is the name of an app. Shouldn't it be title case?

Reply:
oh i forgot to put it in title case, fixed it :)

thank you